Documentation/Books/Manual/SUMMARY.md

```diff
@@ -77,6 +77,7 @@
     * [Limitations](Programs/Arangodump/Limitations.md)
   * [Arangorestore](Programs/Arangorestore/README.md)
     * [Examples](Programs/Arangorestore/Examples.md)
+    * [Parallel Restore Procedure](Programs/Arangorestore/ParallelRestore.md)
```

Comment:
I wonder if it should rather be placed in a Backup & Restore chapter under Administration.

Reply:
Hi, I would say no, as this is specific restore procedure for the tool arangorestore. There might be other restore solutions, therefore I suggest we aggregate things related to arangorestore in that tool page, while we describe other tools and solutions elsewhere. I believe it is important to cite/link the arangorestore parallel procedure in the (general) Backup & Restore chapter - and this has been done. Based on the above, hope you can agree